src/context/AppProvider/AppContext.tsx

```diff
@@ -62,6 +66,12 @@ import { useAppDispatch, useAppSelector } from 'state/store'
  * for some time as reselect does a really good job of memoizing things
  *
  */
+
+const { getOpportunityIds } = opportunitiesApi.endpoints
+const selectUniV2LpIds = getOpportunityIds.select({
```

Comment:
can we move this to opportunities selectors, or define inline?

Reply:
Unfortunately can't move this because of circular deps, tried it and got burned :sadpepe: - improved the syntactic 🍬 in 5a7ba5a93468720e445446f1f9fdc6790dfb6f8b